src/themes/dark-theme-colors.js

```diff
@@ -107,6 +107,8 @@ export default {
     tagPositiveBackground: primitives.positive500,
     tagWarningBackground: primitives.warning500,
     tagNegativeBackground: primitives.negative500,
+    tagRGBGradient: '0, 0, 0',
+    tagRGBGradientSecondary: '255, 255, 255',
```

Comment:
should we use the `hexToRgb` function here too, to keep this file consistent, and to only use hex values here? https://github.com/uber-web/baseui/pull/1130/files#diff-b9a386a5a8a07482d2baf6988558bc63R29